examples/gallery/3d_plots/scatter3d.py

```diff
@@ -39,8 +43,23 @@
 # color_model="+cSetosa,Versicolor,Virginica" to write the discrete color
 # palette "cubhelix" in categorical format and add the species names as
 # annotations for the colorbar
+
+# Please note that by default pandas assigns the categorical number code
+# in a lexical or alphabetical manner to the individual categories.
+# Thus, also the labels for the colorbar have to be given to color_model
+# in alphabetical order.
+# In case you want another or non-alphabetical order, you firstly have
+# to adjust the categorical number code. For handling and manipulating
+# categorical data in pandas you may have a look at:
+# https://pandas.pydata.org/docs/user_guide/categorical.html
```

Comment:
After the revision in 7bcea085a2ffc11bf4a1afe00c74c10d6aa47685, the labels in the `makecpt` call is automatically determined. Do you think we should remove this long paragraph?

Reply:
Maybe we can shorten this paragraph a bit and then shift it to the top (see @seisman's second comment).

Reply:
Sounds good.

Reply:
Done.